client/src/reader/ReaderApp.js

```diff
@@ -35,35 +43,41 @@ const ReaderApp = ({ token, userSettings, getUserLibrary }) => {
   };
 
   return (
-    <div className={styles.container} style={fontStyle}>
-      <Switch>
-        <Route exact path={routes.ROOT} component={AdventureBrowser} />
-        <Route
-          path={routes.AUTH_REDIRECT}
-          render={props => <AuthRedirect rootPath={routes.ROOT} {...props} />}
-        />
-        <Route path={routes.NOT_FOUND} component={NotFound} />
-        <Route
-          path={routes.CREATE_USERNAME}
-          render={props => <CreateUsername rootPath={routes.ROOT} {...props} />}
-        />
-        <AuthRoute
-          exact
-          path={routes.LIBRARY}
-          loginPath={routes.ROOT}
-          component={Library}
-        />
-        <AuthRoute
-          path={routes.READ}
-          loginPath={routes.ROOT}
-          component={Read}
-        />
-        <Route path={routes.COVER} component={Cover} />
-        <Route path={routes.PROFILE} component={Profile} />
-        <Route path={routes.FAQ} component={FrequentlyAskedQuestions} />
-        <Route component={NotFound} />
-      </Switch>
-    </div>
+    <React.Fragment>
+      <div className={styles.container} style={fontStyle}>
+        <Switch>
+          <Route exact path={routes.ROOT} component={AdventureBrowser} />
+          <Route
+            path={routes.AUTH_REDIRECT}
+            render={props => <AuthRedirect rootPath={routes.ROOT} {...props} />}
+          />
+          <Route path={routes.NOT_FOUND} component={NotFound} />
+          <Route
+            path={routes.CREATE_USERNAME}
+            render={props => (
+              <CreateUsername rootPath={routes.ROOT} {...props} />
+            )}
+          />
+          <AuthRoute
+            exact
+            path={routes.LIBRARY}
+            loginPath={routes.ROOT}
+            component={Library}
+          />
+          <AuthRoute
+            path={routes.READ}
+            loginPath={routes.ROOT}
+            component={Read}
+          />
+          <Route path={routes.COVER} component={Cover} />
+          <Route path={routes.PROFILE} component={Profile} />
+          <Route path={routes.FAQ} component={FrequentlyAskedQuestions} />
+          <Route component={NotFound} />
+        </Switch>
+      </div>
+      <Modal />
+      <Toast />
```

Comment:
Had to wrap this shit in a react fragment so I could add the Modal & Toast, so that's why there's some Big Diff Energy going on here.